corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -658,7 +669,10 @@ def update_report(self):
                 matching_data_source.is_deactivated = False
                 matching_data_source.save()
                 tasks.rebuild_indicators.delay(matching_data_source._id)
-
+            changed = self._change_indicators(matching_data_source.configured_indicators)
+            if changed:
+                matching_data_source.save()
+                tasks.rebuild_indicators.delay(matching_data_source._id)
```

Comment:
It would be nice to do a changed or reactivated check in this method the way you did in `create_report` to avoid the possibility of starting two rebuild tasks